client/src/components/graph/overlays/centroidLabels.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable jsx-a11y/mouse-events-have-key-events */
+import React, { PureComponent } from "react";
+
+export default class CentroidLabels extends PureComponent {
+  render() {
+    const {
+      labels,
+      inverseTransform,
+      dilatedValue,
+      dispatch,
+      colorAccessor
+    } = this.props;
+
+    const iter = labels.entries();
+    let pair = iter.next().value;
+    let value;
+    let key;
+
+    const labelSVGS = [];
+    let fontSize = "15px";
+    let fontWeight = null;
+    while (pair) {
```

Comment:
style: why not just use for of?   Would remove all the explicit iterator handling.

Eg., `for (const pair of labels.entries()) {...}`

Reply:
BTW, I saw this pattern used in several other files (eg centroid.js)

Reply:
removed in favor of .forEach()